lib/datadog/core/remote/ext.rb

```diff
@@ -5,7 +5,7 @@ module Core
     module Remote
       module Ext
         ENV_ENABLED = 'DD_REMOTE_CONFIGURATION_ENABLED'
-        ENV_POLL_INTERVAL_SECONDS = 'DD_REMOTE_CONFIGURATION_POLL_INTERVAL_SECONDS'
+        ENV_POLL_INTERVAL_SECONDS = 'DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS'
```

Comment:
This makes this setting somewhat inconsistent with `DD_REMOTE_CONFIGURATION_ENABLED`. I've searched on github and it seems internally we have both `DD_REMOTE_CONFIGURATION_ENABLED` and `DD_REMOTE_CONFIG_ENABLED`...

Maybe worth raising with the remote config folks what they want to do about this?

Reply:
I just asked on their Slack channel